backend/src/main/java/com/woowacourse/levellog/team/dto/TeamDto.java

```diff
@@ -22,6 +22,7 @@ public class TeamDto {
     private String place;
     private LocalDateTime startAt;
     private String teamImage;
+    private Boolean isClosed;
```

Comment:
```suggestion
    @Accessors(fluent = true)
    private boolean isClosed;
```
https://developer-youngjun.tistory.com/18
👀

Reply:
isClosed 를 추가함으로써 API 명세에 변경이 생겼군요.
프론트한테 해당 사항을 꼭 공유해주세요 😇

Reply:
넵!

Reply:
boolean 타입은 @Accessors(fluent = true)를 해주지 않아도 getter의 메소드 명이 isClosed로 만들어 져서 위에서 언급한 문제가 그대로 발생해요😢